Automattic/wp-calypso
# Jetpack Cloud: Add new Backups Upsell, refactor common components from Scan Upsell

#### Changes proposed in this Pull Request

* Add jetpack cloud upset component based on Scan Upsell
* Use new upsell component in scan and backup upsells

![screenshot](https://user-images.githubusercontent.com/2810519/80641136-42ef0e80-8a19-11ea-8e5d-f3dac57eef37.png)


#### Testing instructions

1. navigate to `/backups` for a site without backups
2. verify the backup works and looks as expected
3. repeat with `/scan`
